shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1698,6 +1698,16 @@ void WebContents::DidFinishNavigation(
            frame_process_id, frame_routing_id);
     }
   }
+  content::NavigationEntry* entry = navigation_handle->GetNavigationEntry();
+
+  // If a history entry has been made and the forward/back call has been made,
+  // proceed with setting the new title
+  ui::PageTransition transition =
+      ui::PageTransitionFromInt(navigation_handle->GetPageTransition() &
+                                ui::PAGE_TRANSITION_FORWARD_BACK);
+  if (entry && ui::PageTransitionTypeIncludingQualifiersIs(
+                   ui::PAGE_TRANSITION_FORWARD_BACK, transition))
```

Comment:
Why the int conversion ?

```suggestion
  if (entry &&
      (entry->GetTransitionType() & ui::PAGE_TRANSITION_FORWARD_BACK))
```

Reply:
This stemmed from some logging I was doing before the if statement. This is more efficient though, so thank you.